core/domain/suggestion_registry.py

```diff
@@ -1095,3 +1095,95 @@ def reject(self, reviewer_id, rejection_message):
         SuggestionTranslateContent),
     suggestion_models.SUGGESTION_TYPE_ADD_QUESTION: SuggestionAddQuestion
 }
+
+
+class ReviewerAndSuggestionCounts(python_utils.OBJECT):
+    """Domain object for the ReviewerAndSuggestionCountsModel.
+
+    Attributes:
+        translation_reviewer_counts_per_lang: dict. A dictionary that contains
+            the total number of translation reviewers for each language that
+            translation suggestions are offered in.
+        translation_suggestion_counts_per_lang: dict. A dictionary that
+            contains the total total number of translation suggestions that are
```

Comment:
"total" is repeated.

Also, same comment as above re clearer documentation of keys and values.